README.md

```diff
@@ -100,10 +100,10 @@ Create a `faucet.config.js` next to `package.json`. Here's the one from `demo-we
             webRoot: targetBaseDir
         }
     };
-    
+
 You'll notice that it puts the all processed assets into `./target/classes/static`. That is where Spring Boot looks for static files by default. Pushing it directly into the classes folder allows dynamic reloading later on. An alternative would be going through `generated-resources`.
 
-If you also opt for subdirectories for different kind of assets, you'll have to configure them in your Spring application as well:
+As the above configuration writes the assets into subdirectories you have to tell your Spring application to include those path patterns:
```

Comment:
Please change to: "subdirectories, you have to configure your Spring application to include those path patterns"

I'll fix the Travis build in the meantime.